packages/extension-link/src/link.ts

```diff
@@ -6,6 +6,11 @@ import { autolink } from './helpers/autolink'
 import { clickHandler } from './helpers/clickHandler'
 import { pasteHandler } from './helpers/pasteHandler'
 
+export interface LinkProtocolOptions {
+  scheme: string;
+  optionalSlashSlash?: boolean;
```

Comment:
I'm not sure about the name of this option. How about `optionalSlashes`?

Reply:
I used the argument name from linkify but I'm OK with `optionalSlashes` too